ReactiveCocoa/Swift/Observer.swift

```diff
@@ -56,11 +67,17 @@ public struct Observer<Value, Error: ErrorType> {
 
 extension Observer: ObserverType {
 	/// Puts a `Next` event into the given observer.
+	///
+	/// - parameters:
+    ///   - value: a value sent with the `Next` event
```

Comment:
Aaaand I just realised that this project is tabs not spaces.  My bad, I'll tidy this up on a branch on my fork and will push the rest of these commits up to here when they're all done 😄 

https://www.youtube.com/watch?v=SsoOG6ZeyUI